routes/userRoutes.js

```diff
@@ -22,11 +38,11 @@ router.post('/register', (req, res) => {
 router.post('/login', (req, res) => {
   let { username, password } = req.body;
 
-  Users.findUser({username})
+  Users.findUser({ username })
        .first()
        .then(user => {
          if(user && bcrypt.compareSync(password, user.password)){
-           //req.session.username = user.username;
+           req.session.user = user;
```

Comment:
Typically, even though it's hashed, you probably don't want to store a user's password like this (in a session).

Reply:
Removed password from storage.